_posts/2024-12-12-WildFly35Beta-released.adoc

```diff
@@ -0,0 +1,47 @@
+
+* Standard WildFly now link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19866-promote-mp-rest-client-to-default.html[supports MicroProfile REST Client 4.0, window=_blank]. This was previously supported in WildFly Preview.
+
+* Standard WildFly now link:https://docs.wildfly.org/wildfly-proposals/observability/WFLY-19846-promote-mp-tel-2-to-default.html[supports MicroProfile Telemetry 2.0, window=_blank]. This was previously supported in WildFly Preview.
+
+* Our MicroProfile Reactive Messaging subsystem has added OpenTelemetry tracing integration for link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19835_microprofile_reactive_messaging_otel_with_kafka.html[Kafka connectors, window=_blank] and link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19836_microprofile_reactive_messaging_otel_with_amqp.html[AMQP connectors, window=_blank].
+
+There's plenty of new things beyond MicroProfile area as well, though:
+
+* WildFly now includes link:https://docs.wildfly.org/wildfly-proposals/scripts/WFCORE-6935-system-daemon-scripts.html[supported `systemd` units] in the `bin/systemd` directory, replacing the old, unsupported `docs/contrib/scripts/systemd` files. The new units include support for a managed domain.
+
+* The `jaxrs` subsystem now link:https://docs.wildfly.org/wildfly-proposals/jaxrs/WFLY-13122_Add_Json_Merge_Patch_support.html[provides JSON merge patch support, window=_blank]. This feature allows the client to send a JSON http request with Content-Type "application/merge-patch+json", and the JSON content will be directly merged to the target resource.
+
+* We added link:https://docs.wildfly.org/wildfly-proposals/ee/WFLY-19776_Jakarta_Data_in_standard_WildFly.html[support for Jakarta Data 1.0, window=_blank] to standard WildFly. This was previously link:https://www.wildfly.org/news/2024/10/10/jakarta-data/[included in WildFly Preview 34, window=_blank]; now it is available in standard WildFly as well. This feature is provided at the link:https://docs.wildfly.org/34/Admin_Guide.html#Feature_stability_levels[`preview` stability level, window=_blank].
+
+* WildFly Preview has  a new link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19954_Preview_Support_vertx_feature_pack.html[`vertx` subsystem and Galleon layer], intended to give user's greater control over the configuration of Vert.x instances running in the server. This feature is provided at the link:https://docs.wildfly.org/34/Admin_Guide.html#Feature_stability_levels[`preview` stability level, window=_blank].
+
+* WildFly Glow has also received a new feature to https://github.com/wildfly/wildfly-proposals/pull/627[Introduce spaces to structure discovered Galleon feature-packs], this allows feature packs to be grouped into spaces such as an `experimental` space or a `default` space to reflect the stability of the feature pack and to allow users to select which spaces they want to use.
```

Comment:
The link URL can now be https://docs.wildfly.org/wildfly-proposals/wf-galleon/WFLY-19855_glow_spaces.html.

s/Introduce/introduce

In the link adding ", window=_blank" would be good.